source/server/admin/stats_params.cc

```diff
@@ -64,9 +64,16 @@ Http::Code StatsParams::parse(absl::string_view url, Buffer::Instance& response)
 #else
       response.add("HTML output was disabled by building with --define=admin_html=disabled");
       return Http::Code::BadRequest;
+#endif
+    } else if (format_value.value() == "active") {
+#ifdef ENVOY_ADMIN_HTML
+      format_ = StatsFormat::Active;
+#else
+      response.add("Active HTML output was disabled by building with --define=admin_html=disabled");
+      return Http::Code::BadRequest;
 #endif
     } else {
-      response.add("usage: /stats?format=(html|json|prometheus|text)\n\n");
+      response.add("usage: /stats?format=(html|active|json|prometheus|text)\n\n");
```

Comment:
Is there a more descriptive name than `active`? Without some more context, a user seeing this won't have a good idea of what it means. Maybe `activehtml` or similar, so they at least know it's intended to be used by a browser (not curl/cmdline)?

Reply:
changed to active-html.